docs/quickstarts/excel-custom-functions-quickstart.md

```diff
@@ -0,0 +1,109 @@
+
+- [Node.js](https://nodejs.org/en/) (version 8.0.0 or later)
+
+- [Git Bash](https://git-scm.com/downloads) (or another Git client)
+
+- The latest version of [Yeoman](https://yeoman.io/) and the [Yeoman generator for Office Add-ins](https://www.npmjs.com/package/generator-office). To install these tools globally, run the following command via the command prompt:
+
+    ```
+    npm install -g yo generator-office
+    ```
+
+    > [!NOTE]
+    > Even if you have previously installed the Yeoman generator, we recommend updating your package to the latest version from npm.
+
+## Build your first custom functions project
+
+To start, you'll use the Yeoman generator to create the custom functions project. This will set up your project with the correct folder structure, source files, and dependencies to begin coding your custom functions.
+
+1. Run the following command and then answer the prompts as follows.
+
+    ```
+    yo office
+    ```
+
+    - Choose a project type: `Excel Custom Functions Add-in project (...)`
+
+    - Choose a script type: `JavaScript`
+
+    - What do you want to name your add-in? `stock-ticker`
+
+    ![Yeoman generator for Office Add-ins prompts for custom functions](../images/12-10-fork-cf-pic.jpg)
+
+    The Yeoman generator will create the project files and install supporting Node components.
+
+2. Go to the project folder.
+
+    ```
+    cd stock-ticker
+    ```
+
+3. Trust the self-signed certificate that is needed to run this project. For detailed instructions for either Windows or Mac, see [Adding Self Signed Certificates as Trusted Root Certificate](https://github.com/OfficeDev/generator-office/blob/master/src/docs/ssl.md).  
+
+4. Build the project.
+
+    ```
+    npm run build
+    ```
+
+5. Start the local web server, which runs in Node.js.
+
+    - If you'll be using Excel for Windows to test your custom functions, run the following command to start the local web server, launch Excel, and sideload the add-in:
+
+        ```
+         npm run start
+        ```
+        After running this command, your command prompt will show details about starting the web server. Excel will start with your add-in loaded. If you add-in does not load, check that you have completed step 3 properly.
+
+    - If you'll be using Excel Online to test your custom functions, run the following command to start the local web server:
+
+        ```
+        npm run start-web
+        ```
+
+         After running this command, your command prompt will show details about starting the web server. To use your functions, open a new workbook in Excel Online. In this workbook, you'll need to load your add-in. 
+
+        To do this, select the **Insert** tab on the ribbon and select **Office Add-ins**. Next select **Manage My Add-ins > Upload My Add-in**. Browse for your manifest file and upload it. If your add-in does not load, check you've completed step 3 correctly.
```

Comment:
```suggestion
        To do this, select the **Insert** tab on the ribbon and select **Office Add-ins**. Next, select **Manage My Add-ins > Upload My Add-in**. Browse for your manifest file and upload it. If your add-in does not load, check you've completed step 3 correctly.
```

Reply:
Are you sure that "Office Add-ins" is on the Insert tab? When I look at mine, it offers two choices "Get Add-ins" and "My Add-ins" but not "Office Add-ins."

Reply:
```suggestion
        To do this, select the **Insert** tab on the ribbon and select **Get Add-ins**. In the resulting new window, ensure you are on the **My Add-ins** tab. Next, select **Manage My Add-ins > Upload My Add-in**. Browse for your manifest file and upload it. If your add-in does not load, check you've completed step 3 correctly.
```
I'm still not sure why this is like 5 clicks to just get a thing to sideload, I have always found this very cumbersome to explain. Is that something we should take to the product team and/or UX folk? Good catch btw!

Reply:
Also - numbers in docs. Do we need to spell out three or is it ok to just put the number?

Reply:
@mscharlock You can use the numeral for referring to numbered sections in a list. "Step 3" is ok! (See "Numbers" in the Microsoft Style Writing Guide)